static/.htaccess

```diff
@@ -7,8 +7,21 @@ RewriteRule (.*) https://%{SERVER_NAME}/$1 [R=301,L]
 Options -Indexes -MultiViews
 
 # For linking from our apps - redirect URLs with the current version to URLs with no version
-RewriteRule ^docs/8.2/(.*)$ /docs/$1 [R=302,L]
-RewriteRule ^optimize/3.10.0/(.*)$ /optimize/$1 [R=302,L]
+RewriteRule ^docs/8.3/(.*)$ /docs/$1 [R=302,L]
+RewriteRule ^optimize/3.11.0/(.*)$ /optimize/$1 [R=302,L]
+
+# 8.2 content moves: These were removed from latest version docs, so point them back to previous version for now. 
+#   Remove them in the not-distant future.
+RewriteRule ^docs/guides/update-guide/026-to-100/?$ /docs/8.2/guides/update-guide/026-to-100/ [R=301,L]
+RewriteRule ^docs/guides/update-guide/100-to-110/?$ /docs/8.2/guides/update-guide/100-to-110/ [R=301,L]
+RewriteRule ^docs/guides/update-guide/110-to-120/?$ /docs/8.2/guides/update-guide/110-to-120/ [R=301,L]
+RewriteRule ^docs/guides/update-guide/120-to-130/?$ /docs/8.2/guides/update-guide/120-to-130/ [R=301,L]
+
+# 8.3 content moves
+RewriteRule ^docs/guides/update-guide/connectors/(.*)$ /docs/components/connectors/custom-built-connectors/update-guide/$1 [R=301,L]
+RewriteRule ^docs/guides/update-guide/(.*)$ /docs/self-managed/operational-guides/update-guide/$1 [R=301,L]
+RewriteRule ^docs/self-managed/backup-restore/(.*)$ /docs/self-managed/operational-guides/backup-restore/$1 [R=301,L]
+RewriteRule ^docs/self-managed/troubleshooting/log-levels/(.*)$ /docs/self-managed/operational-guides/troubleshooting/log-levels$1 [R=301,L]
```

Comment:
I've been advocating for us to not maintain rewrite rules for "next" docs lately, but here I see the down-side -- I had to identify all these pages that moved in the next version, and add redirects for them.

I don't think this is wrong or a reason for us to maintain redirect rules for next docs...it is just noteworthy.